cmd/healthbot/counterprobe/counterprobe.go

```diff
@@ -11,7 +11,7 @@ import (
 	logger "github.com/rs/zerolog/log"
 	clientV1 "github.com/textileio/go-tableland/pkg/client/v1"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/metric/instrument/syncint64"
```

Comment:
There's no `go.opentelemetry.io/otel/metric/instrument/syncint64` package now. All instruments can be found in `go.opentelemetry.io/otel/metric/instrument`